matrix_client/api.py

```diff
@@ -241,6 +241,29 @@ def send_message_event(self, room_id, event_type, content, txn_id=None,
             params["ts"] = timestamp
         return self._send("PUT", path, content, query_params=params)
 
+
+    def send_redact_event(self, room_id, event_id, content, txn_id=None, timestamp=None):
+        """Perferm PUT /rooms/$room_id/redact/$event_id/$txn_id/
+
+        Args:
+            room_id(str): The room ID to redact the message event in.
+            event_id(str): The event id to redact.
+            content(dict): The JSON content to send.
```

Comment:
Since the only key that belongs in the body of this request is `reason`,
let's just include `reason` kwarg on the method call signature. Instead of
having a `content` kwarg that will always look like `{"reason": "<text>"}`